test/parallel/test-debugger-list-no-module-wrapper.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const common = require('../common');
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+const fixture = path.join(
+  common.fixturesDir,
+  'debugger-repeat-last.js'
+);
+
+const args = [
+  'debug',
+  `--port=${common.PORT}`,
+  fixture
+];
+
+const proc = spawn(process.execPath, args, { stdio: 'pipe' });
+proc.stdout.setEncoding('utf8');
+
+let stdout = '';
+
+let sentCommand = false;
+let sentEmpty = false;
+let sentExit = false;
+
+proc.stdout.on('data', (data) => {
+  stdout += data;
+  if (!sentCommand && stdout.includes('> 1')) {
+    sentCommand = true;
+    return;
+  }
+
+  if (!sentEmpty) {
+    setImmediate(() => { proc.stdin.write('list(10)\n'); });
+    sentEmpty = true;
```

Comment:
Can you change the variable name to something that is more applicable to this test, like `sentList`?